content/en/docs/started/installing-kubeflow/index.md

```diff
@@ -320,7 +320,7 @@ try out the end-to-end Kubeflow Platform capabilities.
  
 If you want a stable / conservative experience we recommend to use the [latest stable release](https://github.com/kubeflow/manifests/releases): 
 - [**Kubeflow 1.10:**](/docs/releases/kubeflow-1.10/)
```

Comment:
thank you for working on this, please update the kubeflow version as well to 1.10.1

Reply:
From what I can tell the latest release on the `kubeflow/kubeflow` side is `v1.10.0`: https://github.com/kubeflow/kubeflow/releases/tag/v1.10.0